# publish_rate in Hz
publish_rate: 1000.0
# Possible units: [mm, cm, m]
units: mm
# Workspace [width, depth, height] = [x, y, z]
hysteresis: 3.0
workspace:
  width: 80.0
  depth: 80.0
  height: 80.0
  center: [0, 0, 0]
# Position parameters
reference_frame: world
position_ratio: 250
pivot_dist: 5.0
axes_mapping: [x, y, z]
# Force feedback parameters
k_center: 0.15
b_center: 0.003
k_rate: 0.01
b_rate: 0.003
rate_gain: 0.05
vibration:
  a: 1.0
  c: 5.0
  frequency: 30.0
  duration: 0.3
  
# Rotation parameter to synchronize rotation axis
axes_arot_3: [-1, 0, 0]
angle_arot_3: 0.0
axes_arot_2: [0, -1, 0]
angle_arot_2: 0.0
axes_arot_1: [-1, 0, 0]
angle_arot_1: 0.0

# Rotation parameter to synchronize rotation movement
axes_mrot_3: [1, 0, 0]
angle_mrot_3: 0.0
axes_mrot_2: [0, -1, 0]
angle_mrot_2: 0.0
axes_mrot_1: [-1, 0, 0]
angle_mrot_1: 0.0

#~ angle_rotation: 1.570796
